src/components/AddTodo.jsx

```diff
@@ -25,40 +25,50 @@ const AddTodo = (props = (todos, setTodos)) => {
         }));
     };
 
-    const handleCreateTodo = () => {
+    const handleCreateTodo = (event) => {
+        //TODO: Use event.preventDefault()
         axios
             .post(import.meta.env.VITE_TODO_API, data)
-            .then((res) => props.setTodos([...props.todos, res.data]))
+            .then((res) => setTodos([...todos, res.data]))
```

Comment:
you can put an optional chain operator `setTodos?.()` that way if you forget to pass the function as a prop the app wont break.
or 
you can set a default value for the prop like you did  for the todos `{..., setTodos = () => {}}`